src/server/verification/verificationAPI.js

```diff
@@ -44,7 +45,7 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
         await processor.enqueueDisposal(user, enrollmentIdentifier, log)
       } catch (exception) {
         const { message } = exception
-        log.error('delete face record failed:', message, exception, { enrollmentIdentifier, user })
+        logException(log, 'delete face record failed:', message, exception, { enrollmentIdentifier, user })
```

Comment:
dont use logException its not explicit enough
use log.warn and log.error directly

Reply:
should be error

Reply:
1. in that case we'll have code repetition. that because i added a helper method. i suggest to make its name not so abstract (e.g. logVerificationIssue or processVerificationIssue)
2.
here could be some unexpected issue like timeout but also some exception throw by following the logic (e.g. invalid signature, FV is already disposing etc). so we need to use the helper

Reply:
1. please keep it log. that's universal, this way you can easily search for log.error log.warn etc
2. you can use the helper to give you the case and then do log.error or log.warn